### PR TITLE
Added Tests for User Menu

### PR DESCRIPTION
**Pull request by clintonb**
Additionally, scripts used by nearly every page on the site have been moved to application-main.js.

@dsjen @rocha

### CODE REVIEW

File: acceptance_tests/__init__.py [hunk 1]
```diff
@@ -60,7 +60,20 @@ def _test_footer(self):
...
+    def _test_user_menu(self):
+        """
+        Verify the user menu functions properly.
+        """
+        element = self.page.q(css='a.active-user.dropdown-toggle')
+        self.assertTrue(element.present)
+        self.assertEqual(element.attrs('aria-expanded')[0], 'false')
+
+        element.click()
+        self.assertEqual(element.attrs('aria-expanded')[0], 'true')
```
Comment (rocha) [inline acceptance_tests/__init__.py:98]: Should we test if the element is visible too?

**Comment by rocha** [comment]
:+1: It seems to me that we want to test both that the element is visible and that the aria property is updated, but that may not be necessary according to how the element works.

File: acceptance_tests/__init__.py (hunk 1 above)
Comment (clintonb) [inline acceptance_tests/__init__.py:98]: Updated.

**Comment by rocha** [comment]
:rocket:

**Pull request merged**